universum/modules/output/github_output.py

```diff
@@ -0,0 +1,47 @@
+from .terminal_based_output import TerminalBasedOutput, stdout
+
+__all__ = [
+    "GithubOutput"
+]
+
+
+class GithubOutput(TerminalBasedOutput):
+    """
+    GitHub doesn't support nested grouping
+    See: https://github.com/actions/runner/issues/802
```

Comment:
I think, if you did use some official user guide for GitHub Actions, it would be nice to add the link here too, not only that one to a very specific issue.